### DISCUSSION TITLE
Actions

### DISCUSSION THREAD

**Discussion by shuvashish76** [VishnuSanal/Quotes#105] [Ideas]
Setting > `Actions` _(New*)_

![Screenshot_2022-02-12-01-08-34](https://user-images.githubusercontent.com/48226312/153658258-5ec1c9a1-dfcd-47a7-ad1c-e4a711dbfea1.jpeg)
1. Actions > `Home screen swipe actions`
- copy to clipboard
- share as image
- save to gallery
- ask everytime
- add to favourite _(New*)_

2. Actions > `Swipe actions for favourites`
- copy to clipboard
- share as image
- save to gallery
- ask everytime
- delete (fixed for either left or right gesture)

---

screenshorts : refer #103
3. Actions > `Notification buttons action`
- copy to clipboard
- share as image
- save to gallery
- add to favourites
- none/disable

**Comment by VishnuSanal** [comment]
I vote for Favorites and Notifications. But I think changing the behavior of current home screen would be confusing to the users.

**Reply by shuvashish76** [reply]
Ok
1. Settings > `Actions` > `Share button action`
- copy to clipboard
- share as image
- save to gallery
- ask everytime
